.orchestra/support/revng-distributable/install.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Copying root"
+rm -rf "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root"
+mkdir -p "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root"
+
+cd "$ORCHESTRA_ROOT"
+index_files=$(echo share/orchestra/glibc.idx share/orchestra/cmake.idx share/orchestra/boost.idx \
+                   share/orchestra/clang-release.idx share/orchestra/toolchain_*.idx \
+                   | sed 's/ /\n/g' | grep -v host)
+
+EXCLUDE_LIST=$(mktemp -p "${TMP_ROOT}")
+
+while IFS= read -r index_file; do
+  if [ -e "$index_file" ]; then
+    grep -vE '(libgcc_s.so|libc++.so|libc++abi.so|llvm-link|llc)' "$index_file"
+  fi
+done <<< "$index_files" | sed 's|^|/|' > "$EXCLUDE_LIST"
+
+rsync \
+  --archive \
+  --verbose \
+  --progress \
+  --exclude "/revng/" \
+  --exclude "/include/" \
+  --exclude "/lib64/clang/" \
+  --exclude "/lib64/libclang-cpp.so*" \
+  --exclude "/libexec/gcc" \
+  --exclude "/share/binutils-data/" \
+  --exclude "/share/gcc-data/" \
+  --exclude "/share/gdb/" \
+  --exclude "/share/graphviz/" \
+  --exclude "/share/man/" \
+  --exclude "/share/qemu/" \
+  --exclude "/share/revng/qa/" \
+  --exclude "/share/terminfo/" \
+  --exclude "/x86_64-pc-linux-gnu/" \
+  --exclude "/lib64/binutils/" \
+  --exclude "/lib64/gcc/" \
+  --exclude "**/cmake/" \
+  --exclude-from="$EXCLUDE_LIST" \
+  "$ORCHESTRA_ROOT/." \
+  "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root/."
+
+# Manually add PipelineC headers
+mkdir -p "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root/include/revng/PipelineC"
+cp -a "$ORCHESTRA_ROOT"/include/revng/PipelineC/*.h \
+      "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root/include/revng/PipelineC"
+
+cd "$TMP_ROOT/$ORCHESTRA_ROOT/revng"
+
+echo "Creating environment"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/environment" environment
+
+# shellcheck disable=SC2094
+orchestra environment | \
+  grep -E "^export (PATH|PKG_CONFIG_PATH|PYTHONPATH|LIBRARY_PATH)" \
+    >> environment
+
+cat >> environment <<EOF
+unset ORCHESTRA_DOTDIR
+unset ORCHESTRA_ROOT
+EOF
+
+echo "Copying revng-distributable scripts"
+cp -a \
+  "$ORCHESTRA_DOTDIR/support/revng-distributable/revng-update" \
+  "$ORCHESTRA_DOTDIR/support/revng-distributable/revng-system-info" \
+  "root/bin/"
+
+echo "Copying README.md"
+cp -a "$ORCHESTRA_DOTDIR/support/revng-distributable/README.md" .
```

Comment:
Review the README, does it still make sense?

Reply:
Fixed it up, removing mentions of the UI.
However in general it depends on our intentions w.r.t. reusing `revng-distributable` also for local install tarballs or not